modules/account-lib/src/utils/crypto.ts

```diff
@@ -103,3 +105,48 @@ export function isValidPrv(prv: string): boolean {
   }
   return true;
 }
+
+/**
+ * Whether the input is a valid ed25519 private key
+ *
+ * @param {string} prv A hexadecimal private key to validate
+ * @returns {boolean} Whether the input is a valid public key or not
+ */
+export function isValidEd25519Seed(prv: string): boolean {
+  try {
+    const decodedPrv = toUint8Array(prv);
+    return decodedPrv.length === nacl.sign.seedLength;
```

Comment:
this I think is gonna present a major problem if we ever get a prv that is encoded and the same size as our seed. I think we should use whatever helpers or encoding format to gate this if we can

Reply:
Size is not variable AFAIK. The standard size for a seed is 32 bytes, the same as public keys. Secret keys are derived from seeds and they are 64 bytes: https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/#:~:text=Ed25519%20keys%20start%20life%20as,and%20a%20%E2%80%9Cright%20half%E2%80%9D

Reply:
Also: https://cryptobook.nakov.com/digital-signatures/eddsa-and-ed25519

Reply:
oh yeah this looks good. ok my only concern is prvs then so this is dismissed